pilot/pkg/model/validation.go

```diff
@@ -583,10 +583,6 @@ func ValidateSidecar(name, namespace string, msg proto.Message) (errs error) {
 		}
 	}
 
-	if len(rule.Ingress) == 0 && len(rule.Egress) == 0 {
-		return fmt.Errorf("sidecar: missing ingress/egress listeners")
-	}
```

Comment:
what happens if want to convey that "I dont want any ingress listeners at all as my workload only communicates outside"? I also want to convey that (for another workload), "I want all the default listeners".

Reply:
actually, this makes no sense.. this will result in empty sidecar object

Reply:
Removed for now - still debating.